scripts/ci/stats/merged_prs.py

```diff
@@ -84,11 +74,28 @@ def main():
         prj['commits'] = pr.commits
         # The branch we are targeting. main vs release branches.
         prj['base'] = pr.base.ref
+        # defaults
+        prj['review_rule'] = "no"
+        prj['time_rule'] = False
+
+        labels = []
+        for label in pr.labels:
+            labels.append(label.name)
+        prj['labels'] = labels
+
+        reviewers = set()
+        for review in reviews:
+            # get list of all approved reviews
+            if review.user and review.state == 'APPROVED':
+                reviewers.add(review.user.login)
 
-        ll = []
-        for l in pr.labels:
-            ll.append(l.name)
-        prj['labels'] = ll
+        for assignee in pr.assignees:
+            if assignee.login in reviewers:
+                assignee_reviews += 1
+                # was reviewed at least by one assignee
+                prj['reviewed_by_assignee'] = "yes"
+                prj['assignee'] = assignee.login
+                prj['review_rule'] = "yes"
 
         # take first assignee, otherwise we have no assignees and this rule is not applicable
         if pr.assignee:
```

Comment:
could you try and have all check performed against `pr.assignees` and not a mix of `pr.assignee` / `pr.assignees`? This would probably allow to group a few if blocks together too?

Reply:
addressed